### ISSUE TITLE
training_config does not return MetricDefinitions

### ISSUE THREAD

**Issue by shunjd** [aws/sagemaker-python-sdk#1448]
**Describe the bug**
`sagemaker.workflow.airflow.training_config` does not return the `MetricDefinitions` when it is provided in the estimator. However, `tuning_config` does have it.

Related: https://github.com/aws/aws-step-functions-data-science-sdk-python/issues/40

MetricDefinitions in `tuning_config`
https://github.com/aws/sagemaker-python-sdk/blob/aceb8e013848937f6cbb0d7c2f0dffdff1f5aabf/src/sagemaker/workflow/airflow.py#L372

**To reproduce**
```
estimator = sagemaker.estimator.Estimator(
        PCA_IMAGE,
        role=EXECUTION_ROLE,
        train_instance_count=1,
        train_instance_type='ml.c4.xlarge',
        output_path=s3_output_location,
        metric_definitions=[{'Name': 'foo', 'Regex': 'bar=(.*?);'}]
    )

result = training_config(estimator)
print(result)
```

**Expected behavior**
`MetricDefinitions` should be included in the result.

**Screenshots or logs**
N/A

**System information**
A description of your system. Please provide:
- **SageMaker Python SDK version**: latest
- **Framework name (eg. PyTorch) or algorithm (eg. KMeans)**: Estimator/TensforFlow
- **Framework version**:
- **Python version**:
- **CPU or GPU**:
- **Custom Docker image (Y/N)**: N

**Additional context**
N/A

**Comment by ajaykarpur** [comment]
Hi @shunjd, thanks for bringing this to our attention. I've submitted a pull request to address this issue: https://github.com/aws/sagemaker-python-sdk/pull/1453

**Comment by laurenyu** [comment]
looks like this was released in [v1.56.2](https://github.com/aws/sagemaker-python-sdk/releases/tag/v1.56.2)
